### PR TITLE
Switch to YAML, update audacity, wx, ffmpeg

### PR DESCRIPTION
**Pull request by eszlari**

### CODE REVIEW

File: org.audacityteam.Audacity.appdata.xml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+      <li>Convert tapes and records into digital recordings or CDs</li>
+      <li>Edit WAV, AIFF, FLAC, MP2, MP3 or Ogg Vorbis sound files</li>
+      <li>Cut, copy, splice or mix sounds together</li>
+      <li>Change the speed or pitch of a recording</li>
+      <li>Apply a wide range of other effects to audio recordings</li>
+    </ul>
+  </description>
+  <categories>
+    <category>AudioVideo</category>
+    <category>Audio</category>
```
Comment (TingPing) [inline org.audacityteam.Audacity.appdata.xml]: Are these not in the desktop file?

File: org.audacityteam.Audacity.appdata.xml [hunk 2]
```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component>
+  <id>org.audacityteam.Audacity</id>
+  <launchable type="desktop-id">org.audacityteam.Audacity.desktop</launchable>
+  <project_license>GPL-2.0 and CC-BY-3.0</project_license>
```
Comment (TingPing) [inline org.audacityteam.Audacity.appdata.xml]: At a glance source files seem to be `GPL-2.0-or-later`

File: org.audacityteam.Audacity.appdata.xml [hunk 3]
```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (TingPing) [inline org.audacityteam.Audacity.appdata.xml]: In the end any appdata improvements should go upstream.

Comment (eszlari) [reply org.audacityteam.Audacity.appdata.xml]: Of course, I will that. I just wanted to get the update out.

File: org.audacityteam.Audacity.appdata.xml (hunk 1 above)
Comment (eszlari) [reply org.audacityteam.Audacity.appdata.xml]: I forgot that they are copied from the desktop file.

File: org.audacityteam.Audacity.appdata.xml (hunk 2 above)
Comment (eszlari) [reply org.audacityteam.Audacity.appdata.xml]: https://github.com/audacity/audacity/blob/master/LICENSE.txt

**Comment by TingPing** [comment]
bot, build

**Comment by TingPing** [comment]
Fails on arm: https://flathub.org/builds/#/builders/43/builds/279

**Comment by eszlari** [comment]
To fix the arm build, I disabled libmad. This isn't needed anyway, because ffmpeg already provides mp3 import support (I tested that it works).

**Comment by TingPing** [comment]
bot, build

**Pull request merged**